charts/cloudzero-agent/README.md

```diff
@@ -17,255 +17,36 @@ For the latest release, see [Releases](https://github.com/Cloudzero/cloudzero-ch
 
 ## Installation
 
-### Adding Helm Repository Information
+### Easy Install (Most teams will use this!)
 
 To use the chart or a beta version, you must add the repository to Helm. Refer to the [`helm repo`](https://helm.sh/docs/helm/helm_repo/) documentation for command details.
 
-#### 1. Add the Helm Chart Repository
+#### 1. Install the Helm Chart
 
 ```console
 helm repo add cloudzero https://cloudzero.github.io/cloudzero-charts
+helm install cloudzero cloudzero/cloudzero-agent \
+    --set apiKey=<YOUR_CLOUDZERO_API_KEY>\
+    --set clusterName=<YOUR_CLUSTER_NAME>
```

Comment:
it would be really nice to require less - we have a ticket for this [here ](https://cloudzero.atlassian.net/browse/CP-23666)